src-ghc/Pact/Types/Crypto.hs

```diff
@@ -116,21 +115,19 @@ class (ConvertBS (PublicKey a),  Eq (PublicKey a),
 
 
 
-
 --------- HASHING ---------
 
-hashTx :: (H.HashAlgorithm a) => ByteString -> a -> Hash
-hashTx b algo = (Hash . B.convert . H.hashWith algo) b
 
-verifyHashTx :: (H.HashAlgorithm a) => Hash -> ByteString -> a -> Either String Hash
-verifyHashTx h b algo = if hashTx b algo == h
-  then Right h
+verifyHashTx :: PactHash -> ByteString -> Either String Hash
```

Comment:
Why the distinction between PactHash and Hash? Or more specifically, why the conversion from PactHash to Hash upon successful verification?